README.md

```diff
@@ -0,0 +1,50 @@
+# Calculate Journey Variable Payments
+
+## Prerequisites
+
+- [Docker and Docker compose](https://docs.docker.com/get-docker/)
+- [Intellij IDEA](https://www.jetbrains.com/idea/)
+
+## Running locally
+
+You can run the dependencies for the projects with Docker compose:
+
+```bash
+docker-compose up
+```
+
+The command will launch:
+
+- [Localstack](https://github.com/localstack/localstack) (which is used to mock S3)
+- Postgres
+
+Next, you should load the mock data into the Localstack S3.
+
+First, retrieve the ID of the current Localstack container.
+
+```bash
+$ docker ps
+CONTAINER ID        IMAGE                          NAMES
+a51bc3adcd34        localstack/localstack:0.11.5   localstack
+bdae7e94639a        postgres:latest                jpc-database
+```
+
+Attach to the Localstack container with:
+
+```bash
+docker exec -ti <a51bc3adcd34> bash
+```
+
+Load the data with:
+
+```bash
+/docker-entrypoint-initaws.d/setup.sh
+```
+
+Once the command completes, you can close the session.
+
+Head over to Intellij and configure the profile to use Localstack in `Run > Edit configurations`:
```

Comment:
can run from command line with `./gradlew bootRun --args='--spring.profiles.active=localstack'`.  Might not be using Intellij (though lets be honest if you are writing in Kotlin then that is highly unlikely).

Reply:
I will add this in